dissect/target/plugins/os/windows/dpapi/dpapi.py

```diff
@@ -23,12 +23,21 @@ class DPAPIPlugin(InternalPlugin):
 
     SYSTEM_USERNAME = "System"
 
-    def __init__(self, target: Target):
-        super().__init__(target)
-
     def check_compatible(self) -> None:
         if not list(self.target.registry.keys(self.SYSTEM_KEY)):
-            raise UnsupportedPluginError(f"Registry key not found: {self.SYSTEM_KEY}")
+            raise UnsupportedPluginError("Registry key not found: %s", self.SYSTEM_KEY)
+
+    @lru_cache(maxsize=4096)
```

Comment:
Same here, can you initialize this in the `__init__`? There are also no arguments to "LRU" on, so just `@cache` will do.